postgres/setup.py

```diff
@@ -14,6 +14,8 @@
         "testcontainers-core",
         "sqlalchemy",
         "psycopg2-binary",
+        "asyncpg",
+        "pytest-asyncio",
```

Comment:
Let's move this to the [`requirements.in`](https://github.com/testcontainers/testcontainers-python/blob/master/requirements.in) because `pytest-asyncio` is only required for testing.